test/unittests/iotools/converters/genfi_to_bids/test_genfi_to_bids_utils.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+import clinica.iotools.converters.genfi_to_bids.genfi_to_bids_utils as gen
```

Comment:
This function has been moved so the tests will fail:

```suggestion
from clinica.iotools.bids_utils import identify_modality
```